GoogleForCreators/web-stories-wp
# Media Hotlinking: Improve parsing of `Content-Type` header

## Context



## Summary

Fix case where content type returns as `text/vtt; charset=utf-8,`, meaning the hotlink validator fails. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Enable caption hotlinking. 
2. Insert video. 
3. Click Link to file ( captions ). 
4. Try to embed - https://wp.stories.google/stories/how-to-optimize-videos-for-web-stories/tracks/wst024_page2_my_captions.vtt. 
5. See it work correctly. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11134
